gson/pom.xml

```diff
@@ -154,6 +154,12 @@
       <version>${jackson.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>javax.xml.bind</groupId>
+      <artifactId>jaxb-api</artifactId>
+      <version>${jaxb-api.version}</version>
```

Comment:
java xml api was excluded from java distributions starting from Java 9. For jersey to work we need to bring it back.